RecoVertex/Configuration/python/replaceOfflinePrimaryVerticesWithBS.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.MassReplace import massReplaceInputTag as MassReplaceInputTag
+#
+# mass replace OfflinePrimaryVertices with OfflinePrimaryVerticesWithBS (doesn't affect defaults in the source code)
+#
+def massReplaceOfflinePrimaryVerticesToUseBeamSpot(process):
```

Comment:
this is an earlier attempt to achieve the same, but apparently this doesn't affect the defaults created with `fillDescriptions`. Is it foreseen?

Reply:
why not simply to have `offlinePrimaryVertices.vertices="unsortedOfflinePrimaryVertices:WithBS"` ?

also, this could be a processModifier

Reply:
> why not simply to have offlinePrimaryVertices.vertices="unsortedOfflinePrimaryVertices:WithBS" ?

OK, that would work.

> also, this could be a processModifier

what is the functional advantage of a `processModifier`? I was under the impression that such a fixture would be necessary for a "semi-"permanent modification to be made available in release, why in my opinion this would be needed for a one-shot set of samples.

Reply:
>  I was under the impression that such a fixture would be necessary for a "semi-"permanent modification to be made available in release, whle in my opinion this would be needed for a one-shot set of samples.

OK then

Reply:
do we still need this `massReplaceOfflinePrimaryVerticesToUseBeamSpot` method?

Reply:
not for producing relval samples, though it might be nice to have for private studies on top of it.